kubespawner/spawner.py

```diff
@@ -1011,6 +1040,18 @@ def _hub_connect_port_default(self):
         """
     )
 
+    priority_class_name = Unicode(
+        None,
```

Comment:
Perhaps, I'd like that also, but can we do it in a separate PR? I'm only following the convention of all other Unicode fields in the repo.

I was thinking we may break some backward compatibility if we do though, as for example z2jh may assign something a value by writing:
```
# get_config will default to None if somevalue isnt declared
# if we change somevalue to suddenly not accept it being set
# to null then we break things. Are we okay with that?
# This will break if somevalue isn't declared.
c.KubeSpawner.fieldWithoutAllowNoneTrue = get_config('somevalue')
```

Reply:
#187 - related errors that can arise

Reply:
Yes, and that's a problem. I would like to not make it worse by proliferating the pattern, instead doing the right thing for new traitlets.

#187 is not an issue for kubespawner, but a bug in the helm chart (more precisely, the jupyterhub_config.py in the hub image) where it proactively sets values as None when that's invalid, when it shouldn't be setting those values at all.